fuel-vm/src/checked_transaction.rs

```diff
@@ -712,9 +712,9 @@ mod tests {
         let tx = TransactionBuilder::script(vec![], vec![])
             .gas_price(gas_price)
             .gas_limit(gas_limit)
-            .add_unsigned_message_input(rng.gen(), rng.gen(), rng.gen(), input_amount, vec![0xff; 10])
```

Comment:
Instead of replacing all the instances of `rng.gen()`, is it possible to do something like: 
```impl Distribution<SecretKey> for Standard { ... }```
instead? I believe this should allow you to keep the existing syntax in these places.

Reply:
I don't hate this syntax. It's explicit.

Reply:
No. We had that before, but sadly the `elliptic-curve` crate (used by `k256`) [requires](https://docs.rs/elliptic-curve/0.13.4/elliptic_curve/struct.SecretKey.html#method.random) use to accept only rngs with [`CryptoRngCore` trait](https://docs.rs/rand_core/0.6.4/rand_core/trait.CryptoRngCore.html). At least I couldn't figure out how to make `impl Distribution` work with this.